packages/terra-functional-testing/src/commands/validates/screenshot.js

```diff
@@ -18,7 +18,6 @@ const screenshot = (testName, options = {}) => {
   const wrappedOptions = {
     ...options,
     name: testName,
-    updateScreenshots: global.Terra.serviceOptions.updateScreenshots === true,
```

Comment:
Instead of passing down `updateScreenshots` to the global.browser.checkElement, it is now being read from the [Visual Regression service options](https://github.com/cerner/terra-toolkit/pull/583/files#diff-dea71618479daf3d0e21b24c6f7201de0dc18481e558caf0c3aee745a434af06R17).